cmd/auth-provider-gcp/provider/provider_test.go

```diff
@@ -0,0 +1,202 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package provider
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strings"
+	"testing"
+
+	utilnet "k8s.io/apimachinery/pkg/util/net"
+	"k8s.io/cloud-provider-gcp/pkg/gcpcredential"
+	credentialproviderapi "k8s.io/cloud-provider-gcp/pkg/apis/credentialprovider"
+)
+
+const (
+	dummyToken              = "ya26.lots-of-indiscernible-garbage"
+	email                   = "1234@project.gserviceaccount.com"
+	expectedUsername        = "_token"
+	dummyImage = "k8s.gcr.io/pause"
```

Comment:
Formatting, = should be inline with the lines above.

Reply:
Fixed, forgot to run gofmt over this one, sorry about that.